inst/validation/bbr-requirements.yaml

```diff
@@ -106,6 +106,14 @@ CLM-R004:
   description: delete_models() with .tags=NULL
   tests:
   - BBR-CLM-004
+CLM-R005:
+  description: accept '.ctl' file extension when calling new_model
+  tests:
+  - BBR-CLM-005
+CLM-R006:
+  description: accept '.mod' file extension when calling new_model
```

Comment:
Even if you keep multiple test IDs (as I suggest elsewhere, you can roll them up), this can be a single requirement ID with multiple test IDs.